src/chip.tsx

```diff
@@ -27,7 +28,8 @@ const useStyles = createUseStyles(({colors, mq}) => ({
         '& > span': {
             color: colors.textPrimary, // Giving color to text
         },
-
+    },
+    interactive: {
         [mq.supportsHover]: {
```

Comment:
missing cursor pointed in new behaviors

Reply:
checkboxes don't use to have cursor pointer